x-pack/plugins/cases/public/components/status/status_popover_button.tsx

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
This file is the same as `x-pack/plugins/cases/public/components/status/status.tsx` with minor changes